src/app/pages/home/containers/home-page.component.html

```diff
@@ -26,10 +26,15 @@
   </div>
 
   <div
-    class="clinical-container clinical-button"
-    [class.clinical-button-bottom]="showCalendar"
+    class="clinical-container"
+    *ngIf="(hasClinicalTasks | async) || (hasOnDemandTasks | async)"
```

Comment:
is there a way to generalise this? So in the future if we add another type here, we don't have to add those separately.